client/store/index.jsx

```diff
@@ -6,12 +6,14 @@ import { AuthContext } from "../context";
 export const StoreProvider = ({ children, token: initialToken }) => {
   const [token, setToken] = useState(initialToken);
```

Comment:
You should define `email` state here so that other components can access it via `useContext`.

Reply:
I updated it in the way you suggested.